.eslintrc

```diff
@@ -65,6 +65,14 @@
           "render"
         ]
       }
+    ],
+    "camelcase": [
+      "error",
+      {
+        "properties": "never",
+        "ignoreDestructuring": false,
+        "allow": ["^UNSAFE_"]
+      }
```

Comment:
Maybe this could be added to `eslint-config-skyscanner` also?

Reply:
I've raised this for `eslint-config-skyscanner`, just in case: https://github.com/Skyscanner/eslint-config-skyscanner/pull/41